src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -96,7 +96,13 @@ public static void updateRepoConfig(RepoConfiguration config) {
         Path configJsonPath =
                 Paths.get(config.getRepoRoot(), REPOSENSE_CONFIG_FOLDER, REPOSENSE_CONFIG_FILE).toAbsolutePath();
 
-        if (config.isStandaloneConfigIgnored() || !Files.exists(configJsonPath)) {
+        if (!Files.exists(configJsonPath)) {
+            logger.info(String.format("Standalone config file does not exist in %s.", config.getLocation()));
+            return;
+        }
+
+        if (config.isStandaloneConfigIgnored()) {
+            logger.info(String.format("Standalone config file has been ignored in %s.", config.getLocation()));
```

Comment:
Maybe this phrasing is better: `String.format("Ignoring standalone config file in %s.", config.getLocation())`